### PR TITLE
Add v2.0.0-beta release notes

### PR DESCRIPTION
**Pull request by denyeart**
Add release notes for v2.0.0-beta release.

### CODE REVIEW

File: release_notes/v2.0.0-beta.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+The 'Raft' consensus type was introduced in v1.4.1 and has become the preferred
+production consensus type.  There is a documented and tested migration path from
+Kafka to Raft, and existing users should migrate to the newer Raft consensus type.
+For compatibility with existing deployments, Kafka is still supported,
+but may be removed entirely in a future release.
+
+**FAB-7559: Support for specifying orderer endpoints at the global level
+in channel configuration is deprecated.**
+
```
Comment (yacovm) [inline release_notes/v2.0.0-beta.md:163]: Perhaps add: "To fit scenarios where orderers are run by different organizations, you should specify orderer endpoints at the orderer organization level, which uses only the TLS CA certificates of that organization, in contrast to the global channel level endpoints which use the aggregation of all orderer TLS CA certificates across all orderer organizations"

**Review by yacovm — COMMENTED** [review]
added a comment

**Comment by lindluni** [comment]
lgtm

File: release_notes/v2.0.0-beta.md (hunk 1 above)
Comment (denyeart) [reply release_notes/v2.0.0-beta.md:163]: done

**Review by yeasy — APPROVED** [review]

**Review by sykesm — APPROVED** [review]

**Comment by denyeart** [comment]
Addressed comments. Unit test flake, will force merge.

**Pull request merged**